voice/src/main/kotlin/streams/Streams.kt

```diff
@@ -0,0 +1,40 @@
+package dev.kord.voice.streams
+
+import dev.kord.common.annotation.KordVoice
+import dev.kord.common.entity.Snowflake
+import dev.kord.voice.AudioFrame
+import dev.kord.voice.udp.AudioPacket
+import kotlinx.coroutines.flow.Flow
+
+
+/**
+ * A representation of receiving voice through Discord and different stages of processing.
+ */
+@KordVoice
+abstract class Streams {
+    /**
+     * An encryption key used for decryption of Discord packets.
+     */
+    internal abstract var key: ByteArray?
```

Comment:
abstract class with internal abstract member cannot be inherited outside of voice module

Reply:
was a bit too easy i guess :p